src/PKSim.Core/Services/GlobalPKAnalysisTask.cs

```diff
@@ -319,17 +319,23 @@ private bool isMultipleOral(Simulation simulation, Compound compound)
 
       private static bool isOral(ISchemaItem schemaItem)
       {
+         if (schemaItem == null)
+            return false;
+
          return schemaItem.ApplicationType == ApplicationTypes.Oral;
       }
 
       private static bool isIntravenous(ISchemaItem schemaItem)
       {
+         if (schemaItem == null)
+            return false;
+
          return schemaItem.ApplicationType == ApplicationTypes.Intravenous || schemaItem.ApplicationType == ApplicationTypes.IntravenousBolus;
       }
 
       private ApplicationType applicationTypeFor(Simulation simulation, Compound compound)
       {
-         var numberOfApplications = schemaItemsFrom(simulation, compound).Count;
+         var numberOfApplications = simulation.AllApplicationParametersOrderedByStartTimeFor(compound.Name).Count;
```

Comment:
This is the only relevant difference, correct?
For a metabolite (which is not administered) number of applications was zero before; now it is number of applications of its parent drug, right?

By the way, I would rename the function `AllApplicationParametersOrderedByStartTimeFor `to something like `AllOrderedApplicationStartTimes`. 
Because in `AllApplicationParameters `I would expect also all other application parameters like e.g. DrugMass, InfusionTime etc. (I had to look into the implementation of the function to understand what it does)

Reply:
ok but this is in core so not directly relevant to this problem. Please create an issue for this

>For a metabolite (which is not administered) number of applications was zero before; now it is number of applications of its parent drug, right?

yes

Reply:
Issue created: https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/1494
PR looks good for me